pkg/cloudprovider/providers/gce/gce_loadbalancer.go

```diff
@@ -69,7 +76,13 @@ func (gce *GCECloud) EnsureLoadBalancer(clusterName string, apiService *v1.Servi
 		return nil, err
 	}
 
-	loadBalancerName := cloudprovider.GetLoadBalancerName(apiService)
+	loadBalancerName := gce.getLoadBalancerName(apiService)
+
+	annotations := make(map[string]string)
+	annotations["currentLB"] = loadBalancerName
+	apiService.SetAnnotations(annotations)
```

Comment:
Yes. I will change it. I was expecting a function like AppendAnnotation() but I could not find such. I will change it in other way.